src/mito2/src/worker/request.rs

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Worker requests.
+
+use std::time::Duration;
+
+use store_api::storage::{ColumnId, CompactionStrategy, RegionId};
+use tokio::sync::oneshot::{self, Receiver, Sender};
+
+use crate::error::Result;
+use crate::metadata::ColumnMetadata;
+
+/// Options that affect the entire region.
+///
+/// Users need to specify the options while creating/opening a region.
+#[derive(Debug)]
+pub struct RegionOptions {
```

Comment:
Implement ser/de for this struct? It's likely to be stored somewhere

Reply:
`CompactionStrategy` doesn't implement ser/de